tests/Command/RecreateMasterKeyTest.php

```diff
@@ -197,6 +197,7 @@ function($path) {
 				->willReturn(true);
 
 			global $outputText;
+			global $reloginText;
```

Comment:
don't use globals but `use (&$reloginText)` instead in the closure

Reply:
Done